README.md

```diff
@@ -15,31 +15,36 @@ Certificate authentication requires the following files:
 * **username.key** cfr [HowTo Get client access](#howto-get-client-access) !! Your private key **must** remain **secret**,
 * **username.crt** is located in [cert/pki/issued](https://github.com/jenkins-infra/openvpn/tree/master/cert/pki/issued), once an administrator signs  your request and publish it.
 
-```text
-client
-remote vpn.jenkins.io 443
-ca "~/.cert/jenkins/ca.crt"
-cert "~/.cert/jenkins/username.crt"
-key "~/.cert/jenkins/username.key"
-auth-user-pass
-dev tun
-proto tcp
-nobind
-auth-nocache
-script-security 2
-persist-key
-persist-tun
-user nobody
-group nobody
-```
-
-**With network manager client, you must enable the option :**
+**With NetworkManager client, you must enable the option :**
 
 `Use this connection only for resources on its network`
 
 ### DNS Problems
 
-If you are having issues connecting to resources behind the VPN, but the VPN appears to be working correctly, check your DNS settings.  Some providers seem to filter out requests to the zone.  To test, try `dig release.ci.jenkins.io`.
+If you are having issues connecting to resources behind the VPN, but the VPN appears to be working correctly, check your DNS settings.  Some providers seem to filter out requests to the zone.  To test, try `dig release.ci.jenkins.io`, you should get something like this:
+<details><summary>dig output</summary>
+```
+; <<>> DiG 9.10.6 <<>> release.ci.jenkins.io
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 13457
+;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 1220
+;; QUESTION SECTION:
+;release.ci.jenkins.io.         IN      A
+
+;; ANSWER SECTION:
+release.ci.jenkins.io.  3600    IN      CNAME   private.aks.jenkins.io.
+private.aks.jenkins.io. 3600    IN      A       10.0.2.5
+
+;; Query time: 80 msec
+;; SERVER: 192.168.1.254#53(192.168.1.254)
+;; WHEN: Tue Oct 12 20:49:59 CEST 2021
+;; MSG SIZE  rcvd: 92
+```
+</details>
```

Comment:
```suggestion

</details>
```
Since it's a markdown source file, you need to add empty lines between blocks to avoid breaking the final